### ISSUE TITLE
Multiplayer Channels

### ISSUE THREAD

**Issue by Shyclops** [tryashtar/minecraft-stuff#15]
Multiplayer interaction where stations with the same dye color show the same items to all players on server
I apologize if this is already how it works and I'm unaware, unfortunately I only have 1 Minecraft account so was not able to verify if this is existing behaviour

**Comment by MagmaSlime123** [comment]
I think this exists but I may be wrong.

**Comment by tryashtar** [comment]
Yeah, the dye combination inventories shouldn't be specific to any one player
